components/brave_shields/browser/tracking_protection_service.cc

```diff
@@ -54,6 +54,14 @@ void TrackingProtectionService::Cleanup() {
   tracking_protection_client_.reset();
 }
 
+std::vector<std::string> TrackingProtectionService::addFirstPartyTrackers() {
```

Comment:
Method names should be capitalized
These two methods don't really make sense to me. Why do we have add/remove instead of just providing a list?